api/sql/poisSQL/initialize.sql

```diff
@@ -0,0 +1,9 @@
+DROP TABLE IF EXISTS POIs CASCADE;
+
+CREATE TYPE poi_type_enum AS ENUM ('Trailhead', 'Trail Location')
```

Comment:
missing a semicolon (also one on line 9? but i think that one is fine cause it's the end of thje file)